### PR TITLE
Don't show empty tooltips

### PR DESCRIPTION
**Pull request by muffinmad**
Also make tooltip hidden on creation.
Fixing [this issue](https://github.com/mootools/mootools-more/issues/1298)

### CODE REVIEW

**Comment by SergioCrisostomo** [comment]
@muffinmad nice work!

:+1:

File: Source/Interface/Tips.js [hunk 1]
```diff
@@ -170,15 +172,22 @@ this.Tips = new Class({
 		clearTimeout(this.timer);
 		this.timer = (function(){
 			this.container.empty();
-
+			var showTip = !this.options.hideEmpty;
```
Comment (anutron) [inline Source/Interface/Tips.js]: Not a blocker for shipping this, but you declare this variable just to reference it once in a conditional; I'd remove this line and reference the option directly on line 186.

File: Source/Interface/Tips.js [hunk 2]
```diff
@@ -170,15 +172,22 @@ this.Tips = new Class({
...
+					this.fill(div, content);
+					showTip = true;
+				}
 			}, this);
-			this.show(element);
+			if (showTip){
+				this.show(element);
+			} else {
+				this.hide(element);
+			}
```
Comment (anutron) [inline Source/Interface/Tips.js]: minor style thing; MooTools style is to put these on one line:
```javascript
if (true) something();
else somethingElse();
```
only if either of the conditional statements are > 1 line do we add braces (in which case we add braces to both clauses, even if one is one line).

At least, that used to be the convention. Again, not a blocker on shipping it.

File: Source/Interface/Tips.js (hunk 1 above)
Comment (anutron) [inline Source/Interface/Tips.js]: Ignore this. I see line 183...

**Comment by suaritma79** [comment]
sadece koşullu ifadelerden herhangi biri> 1 satır ise küme ayraçları ekleriz (bu durumda her iki cümleye de, biri tek satır olsa bile küme ayraçları ekleriz). https://www.pasifiklife.com/

**Comment by kimnga3338** [comment]
nice

**Pull request merged**